src/main/python/pgshovel/replication/streams/kafka.py

```diff
@@ -29,7 +43,7 @@ def consume(self, state):
         # hard we don't really need to deal with it right now.
         assert len(consumer.offsets) is 1
 
-        if state.HasField('stream_state'):
+        if state.stream_state.HasField('consumer_state'):
```

Comment:
I made this change since looking for `consumer_state` was more specific and there could be a chance that `stream_state.transaction_state` could be set but not `stream_state.consumer_state`, which would cause this `if` statement to evaluate to `True` and the code block to run, using the default value of `0` for `state.stream_state.consumer_state.offset` which would be bad.